pkg/lorry/plugin/proto/db_plugin.proto

```diff
@@ -0,0 +1,256 @@
+/*
+Copyright (C) 2022-2023 ApeCloud Co., Ltd
+
+This file is part of KubeBlocks project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+syntax = "proto3";
+package plugin.v1;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/apecloud/kubeblocks/pkg/lorry/plugin";
+
+extend google.protobuf.FieldOptions {
+  // Indicates that a field MAY contain information that is sensitive
+  // and MUST be treated as such (e.g. not logged).
+  bool kb_secret = 1059;
+}
+
+message GetPluginInfoRequest {
+  map<string, string> metadata = 1;
+}
+
+message GetPluginInfoResponse {
+  // The name of the plugin. It MUST be 63
+  // characters or less, beginning and ending with an alphanumeric
+  // character ([a-z0-9A-Z]) with dashes (-), dots (.), and
+  // alphanumerics between. This field is REQUIRED.
+  string name = 1;
+
+  string db_type = 2;
```

Comment:
what's this field used for? is there any difference with name?  is there any relationship with servicetype?

Reply:
This field is used to indicate which service the plugin is handling, primarily for logging and debugging purposes.